client.go

```diff
@@ -227,7 +227,7 @@ func NewJSONRequest(ctx context.Context, method, url string, data any,
 	return
 }
 
-// RespJSON unmarshals the response body into a new R and closes the body afterwards.
+// RespJSON unmarshals the response body into a new R and closes the body afterward.
```

Comment:
```suggestion
// RespJSON unmarshals the response body into a new 'R' and then closes the body.
```


Refer https://www.thoughtco.com/afterwards-and-afterword-1689292:

There's no difference in meaning between "afterward" and "afterwards", but the two are generally used in separate contexts. In American English, "afterward" is more common, whereas in British English and Canadian English, "afterwards" is.

So I just rephrase it slightly.